### PR TITLE
ntsync5: rebase patches for cebad42f

### PR DESCRIPTION
**Pull request by Artewar67**

### CODE REVIEW

File: wine-tkg-git/wine-tkg-patches/misc/fastsync/ntsync5-staging-protonify.patch [hunk 1]
```diff
@@ -831,29 +831,29 @@ index edfeb03..0000000
 -
```
Comment (kangtastic) [inline wine-tkg-git/wine-tkg-patches/misc/fastsync/ntsync5-staging-protonify.patch:831]: Hi, thanks for this and your past contributions.

The next few lines here revert changes in #1293 and breaks builds on staging again.

File: wine-tkg-git/wine-tkg-patches/misc/fastsync/ntsync5-staging.patch [hunk 2]
```diff
@@ -831,29 +831,29 @@ index edfeb03..0000000
 -
```
Comment (kangtastic) [inline wine-tkg-git/wine-tkg-patches/misc/fastsync/ntsync5-staging.patch:831]: Same here, breaks builds on staging.

**Comment by l12436** [comment]
There are seems missing some extra modify in mainline cause wineserver crash after patch
This is how I fixed.
Do not if correct, but it seems working

[fix-fastsync.diff.gz](https://github.com/user-attachments/files/17515043/fix-fastsync.diff.gz)

**Comment by Artewar67** [comment]
With staging I did not have this commit https://github.com/Frogging-Family/wine-tkg-git/pull/1293 and therefore these patches were written without it

While I'm dealing with Steam and its crashes, I don't recommend merging this request for now.

**Comment by kangtastic** [comment]
Yes, looking over the most recent Wine MR that broke our ntsync5 patch:

https://gitlab.winehq.org/wine/wine/-/merge_requests/6630/

Enough has changed that it might not be so simple this time.

**Comment by Artewar67** [comment]
Looks like I was able to fix the steam. @l12436 Can you test it with this patch?
[0001-fix-ERROR_INVALID_HANDLE-for-steam-with-ntsync-patch.mypatch.zip](https://github.com/user-attachments/files/17524307/0001-fix-ERROR_INVALID_HANDLE-for-steam-with-ntsync-patch.mypatch.zip)

**Comment by Artewar67** [comment]
Now the error of applying a patch for staging no longer appears and a fix for steam has been added

**Comment by Tk-Glitch** [comment]
Thanks Artewar67 and kangtastic!
:frog: :heart: :frog: :heart:

**Comment by l12436** [comment]
The patch that merged is missing some modify I added. three more no_fast_sync for those who use `struct object_ops`.
That will cause wineserver crash

**Pull request merged**